resources/provider/provider.go

```diff
@@ -196,6 +198,8 @@ func Provider() *provider.Provider {
 			"iam.server_certificates":                 iam.IamServerCertificates(),
 			"iam.users":                               iam.IamUsers(),
 			"iam.virtual_mfa_devices":                 iam.IamVirtualMfaDevices(),
+			"inspector.findings":                      inspector.Findings(),
+			"inspector2.findings":                     inspector2.Findings(),
```

Comment:
Should these be under the same service? Because they are the same service, but different versions...

```suggestion
			"inspector.findings":                      inspector.Findings(),
			"inspector.v2_findings":                     inspector2.Findings(),
```

Reply:
I'd argue that it's the same as with waf/elb